content/en/news/releases/1.13.x/announcing-1.13.6/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Announcing Istio 1.13.6
+linktitle: 1.13.6
+subtitle: Patch Release
+description: Istio 1.13.6 patch release.
+publishdate: 2022-07-25
+release: 1.13.6
+aliases:
+- /news/announcing-1.13.6
+---
+
+This release contains bug fixes to improve robustness.
+This release note describes what's different between Istio 1.13.5 and 1.13.6.
+
+FYI, [Go 1.18.4 has been released](https://groups.google.com/g/golang-announce/c/nqrv9fbR0zE),
+which includes 9 security fixes. We recommend you to upgrade to this newer Go version.
```

Comment:
Not sure what the message really means since we have no idea if they user Go or not.In the past we haven't done this, but maybe the PSWG would like this going forward.

Reply:
This is suggested in the PSWG meeting today.

Reply:
Based on today's PSWG meeting, Istio itself is fine; we recommend a user to upgrade to this newer go version as a FYI. @jacob-delgado @GregHanson WDYT?

Reply:
@jacob-delgado @GregHanson Any thoughts on this? It does seem strange to tell the user to update their Go version when it makes no difference to our images. Many may not have Go installed or know little about it. Even the Go announce doesn't really say anything about updating, just that a new release is available. We could remove the last sentence or maybe change to something like `We recommend you to upgrade to this newer Go version if you are using Go locally.`

Reply:
Also 1.13 is built with Go 1.17.12. Should this message be changed to reflect the update for that release.

Reply:
Changed to "We recommend you to upgrade to this newer Go version if you are using Go locally."